dns_nohost_me.md

```diff
@@ -1,6 +1,6 @@
 # Nohost.me domains
 
-In order to make self-hosting as accessible as possible, YunoHost offers a *free*
+In order to make self-hosting as accessible as possible, the YunoHost Project provides a *free*
```

Comment:
Perhaps linking to https://github.com/YunoHost/doc/pull/994/files#diff-ab1b71fc907eef6a6ae61cecfc19728aR6 after mentioning "*free*" would also help direct people to the section where it is explained that this is funded by the project.